sys/socket.txt

```diff
@@ -268,6 +274,12 @@ ifreq_SIOCGIFINDEX {
 	pad		array[const[0, int8], 20]
 } [packed]
 
+
+ifreq_SIOCETHTOOL {
+	ifr_ifrn	devname
+	ifru_data	ptr[inout, ethtool_cmd_u]
```

Comment:
I added a padding of 16 bytes(16+8+16 = 40) supposing the kernel is 64bits else should be 20.